### PR TITLE
[zos-console] Added method to get a command response

### LINKED ISSUE #78: [zos-console] Add method to get a command response
Add a method to the Python zos-console SDK to call the z/OSMF REST API `GET /zosmf/restconsoles/consoles/<console-name>/solmsgs/C<key-number>`

* z/OSMF REST API documentation: https://www.ibm.com/docs/en/zos/2.5.0?topic=services-get-command-response
* Python zos-console SDK in the "next" branch: https://github.com/zowe/zowe-client-python-sdk/tree/next/src/zos_console
* Node.js zos-console SDK for comparison: https://github.com/zowe/zowe-cli/blob/master/packages/zosconsole/src/CollectCommand.ts

### PR DESCRIPTION
**Pull request by aadityasinha-dotcom**
Added a method to call the z/OSMF REST API `GET /zosmf/restconsoles/consoles/<console-name>/solmsgs/C<key-number>`
Signed-off-by: Aaditya Sinha [aadityasinha009@gmail.com](mailto:aadityasinha009@gmail.com)
Resolves #78

### CODE REVIEW

File: tests/unit/test_zos_console.py [hunk 1]
```diff
@@ -19,4 +20,11 @@ def setUp(self):
...
         """Created object should be instance of Console class."""
         console = Console(self.session_details)
-        self.assertIsInstance(console, Console)
\ No newline at end of file
+        self.assertIsInstance(console, Console)
+
+    @mock.patch('requests.Session.send')
+    def test_get_response_should_return_messages(self, mock_send_request):
+        """Getting z/OS Console response messages on sending a response key"""
+        mock_send_request.return_value = mock.Mock(headers={"Content-type": "application/json"}, status_codes=200)
```
Comment (t1m0thyj) [inline tests/unit/test_zos_console.py:28]: Please change `status_codes` -> `status_code` to fix the failing test 🙂

**Review by t1m0thyj — CHANGES_REQUESTED** [review]

File: tests/unit/test_zos_console.py (hunk 1 above)
Comment (aadityasinha-dotcom) [reply tests/unit/test_zos_console.py]: done

**Review by t1m0thyj — APPROVED** [review]

File: tests/unit/test_zos_console.py [hunk 2]
```diff
@@ -19,4 +20,11 @@ def setUp(self):
     def test_object_should_be_instance_of_class(self):
         """Created object should be instance of Console class."""
         console = Console(self.session_details)
-        self.assertIsInstance(console, Console)
\ No newline at end of file
+        self.assertIsInstance(console, Console)
+
+    @mock.patch('requests.Session.send')
+    def test_get_response_should_return_messages(self, mock_send_request):
```
Comment (aadityasinha-dotcom) [inline tests/unit/test_zos_console.py:26]: @zFernand0 added the unit tests. Is there anything which is left?

**Review by zFernand0 — APPROVED** [review]
Happy new year! 😋 

Hey, thanks for addressing all of the comments.
And sorry this took so long for me to review.

File: tests/unit/test_zos_console.py (hunk 2 above)
Comment (zFernand0) [reply tests/unit/test_zos_console.py:26]: nothing I can think of
LGTM! 😋

**Pull request merged**